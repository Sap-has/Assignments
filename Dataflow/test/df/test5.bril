@main {
  x: int = const 10;
  cond: bool = const true;
  br cond .left .right;
.left:
  y: int = add x x;
  jmp .merge;
.right:
  y: int = mul x x;
  jmp .merge;
.merge:
  z: int = add y x;
  print z;
}