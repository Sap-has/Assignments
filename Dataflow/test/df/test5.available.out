b1:
  in:  ∅
  out: ∅
left:
  in:  ∅
  out: add x x
right:
  in:  ∅
  out: mul x x
merge:
  in:  ∅
  out: add y x
